src/mmw/apps/modeling/tasks.py

```diff
@@ -163,12 +176,30 @@ def fn(input):
         measure, code = input
         return {
             'measure': measure,
-            'load': model_output['modified'][code]
+            'load': model_output['modified'][code] * 0.453592  # Lbs to Kg.
```

Comment:
Consider replacing this with a constant declared at the top. That would make it easier to adjust the resolution if we ever need to, and also better communicate the meaning of this (otherwise) magic number.